docs/infrastructure/VPS_SSH.md

```diff
@@ -39,4 +39,37 @@ If you encounter problems with the connection. Use the debug mode `ssh -vvv <hos
 
 
 ## Tunneling with WinSCP
-If using winscp, you will find detailed information here: https://winscp.net/eng/docs/ui_login_tunnel
\ No newline at end of file
+
+WinSCP is a GUI that makes it very easy to inspect, edit and transfer files on the webserver. Instructions for setting this up on a hypothetical server from the CiTG faculty are provided below. General documentation on tunneling with WinSCP are here: https://winscp.net/eng/docs/ui_login_tunnel
+
+_These instructions were tested with an existing `id_ed25519` key, assume you already have WinSCP installed and can modify a text file on the server in your user directory using the terminal._
+
+Do the following:
+
+1. log in to the server and add your public key to the file `/home/rlanzafame/.ssh/authorized_keys`
```

Comment:
'as exlained above', this only needs to be done once

Reply:
May I suggest to not use your netID in the examples? It would be more intuitive to provide example using placeholders for information that depends on the user. e.g. `/home/<user-name>/.ssh/`

Reply:
oops, i missed that one. i will also add instructions for sudo rights in winscp, since i just solved that one also :)